phoenix-core/src/main/java/org/apache/phoenix/jdbc/PhoenixStatement.java

```diff
@@ -430,12 +429,12 @@ public Integer call() throws SQLException {
                             }
                             //Force update cache and retry if meta not found error occurs
                             catch (MetaDataEntityNotFoundException e) {
-                                if(doRetryOnMetaNotFoundError && e.getTableName()!=null){
-                                    if(LOGGER.isDebugEnabled())
-                                        LOGGER.debug("Reloading table "+ e.getTableName()
-                                                +" data from server");
-                                    if(new MetaDataClient(connection).updateCache(connection.getTenantId(),
-                                        e.getSchemaName(), e.getTableName(), true).wasUpdated()){
+                                if (doRetryOnMetaNotFoundError && e.getTableName() != null) {
+                                    if (LOGGER.isDebugEnabled()) {
+                                        LOGGER.debug("Reloading table " + e.getTableName() + " data from server");
```

Comment:
same here

Reply:
> same here

phoenix code have more this ways that not placeholder in log. I will solved this in another issues.